src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -4871,7 +4871,7 @@ kds_custat_show(struct device *dev, struct device_attribute *attr, char *buf)
 		for (;idx < (exec->num_cus + exec->num_sk_cus); ++idx) {
 			sz += sprintf(buf+sz, "CU[@0x0] : %d status : %d name : %s\n",
 				      ert_cu_usage(xert, idx),
-				      ert_cu_status(xert, idx) ? AP_START : AP_IDLE,
+				      ert_cu_status(xert, idx),
```

Comment:
Did you test this change on hard CU status? This seems to break the protocol of hard CU status reporting.